openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0.md

```diff
@@ -56,17 +56,22 @@ This specification uses the terms "Holder", "Issuer", "Verifier", and "Verifiabl
 
 The following aspects are in scope of this interoperability profile:
 
-* Protocol for issuance of the Verifiable Credentials (can be both remote and in-person) (OID4VCI)
-* Protocol for online presentation of Verifiable Credentials (can be both remote and in-person) (OID4VP)
+* Profile of OID4VCI to issue IETF SD-JWT VCs (can be both remote and in-person), including
+  * Wallet Attestation
+* Profile of OID4VP without using W3C Digital Credentials API [@w3c.digital_credentials_api] to present IETF SD-JWT VCs
+* Profile of OID4VP over W3C Digital Credentials API to present 
+  * IETF SD-JWT VCs 
+  * ISO mdocs
```

Comment:
the "OID4VP without using W3C Digital Credentials API" language feels awkward. I *think* this conveys the same thing.

```suggestion
* Profile of OID4VP to present IETF SD-JWT VCs
* Profile of OID4VP over the W3C Digital Credentials API [@w3c.digital_credentials_api] to present 
  * IETF SD-JWT VCs 
  * ISO mdocs
```